pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[tool.poetry]
+name = "auto-pr"
+version = "1.0.0"
+description = "Perform bulk updates across repositories"
+license = "Apache-2.0"
+
+authors = ["GetYourGuide GmbH"]
+
+readme = "README.md"
+repository = "https://github.com/getyourguide/auto-pr"
+homepage = "https://github.com/getyourguide/auto-pr"
+
+packages = [
+    { include = "autopr" },
+]
+
+[tool.poetry.scripts]
+auto-pr = "autopr:main"
+
+[tool.poetry.dependencies]
+python = "^3.8"
+marshmallow = "*"
+marshmallow-dataclass = "*"
+click = "*"
+pygithub = "*"
+pyyaml = "*"
```

Comment:
Thought about this again. We should probably set specific versions here as the package generated with `poetry build` does not take the lockfile into account, it only declares the direct dependencies as specified here.